openapi/paths/subscriptions@{id}@upcoming-invoice.yaml

```
@@ -0,0 +1,31 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Orders
+  summary: Retrieve subscription order's upcoming invoice
```

nitpick: where possible, we should avoid using possessives `'s`

I think we can also remove `subscription order` (this is described in the description)
```suggestion
  summary: Retrieve an upcoming invoice
```